reference/activity_manager/widget_extension_points.rst

```diff
@@ -0,0 +1,82 @@
+Completeness Widget
+===================
+
+Overview
+________
+
+With the Activity Manager feature comes a widget in the dashboard. The goal of this widget is to display the
+completeness for contributors and owner. Displayed data are contextual. That means that the completeness is not the same
+for each contributor and for the owner. Contributor sees completeness in terms of their own permissions. The owner sees
+the completeness of all the project perimeter.
```

Comment:
> the real full completeness of family requirements

Or I don't know, but "the completeness of all the project perimeter" is not clear to me :)